packages/d/devextreme-aspnet-data.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "devextreme-aspnet-data",
+  "filename": "js/dx.aspnet.data.js",
```

```suggestion
  "filename": "dx.aspnet.data.js",
```
You're using basePath `js`, so the files are pulled up from that subdirectory into the root :)

---

Thank you very much :)